src/setuptools_scm/file_finder_git.py

```diff
@@ -27,6 +27,9 @@ def _git_toplevel(path):
             # ``cwd`` is absolute path to current working directory.
             # the below method removes the length of ``out`` from
             # ``cwd``, which gives the git toplevel
+            assert cwd[len(cwd) - len(out):].replace('\\','/') == out
```

Comment:
please use the `str.endswith` method for better readability

also please take a look at https://www.python.org/dev/peps/pep-0616/#specification for a simpler expression of the suffix dropping

Reply:
> please use the str.endswith method for better readability

:+1: 

> also please take a look at https://www.python.org/dev/peps/pep-0616/#specification for a simpler expression of the suffix dropping

ah! negative indexing, let me do.

Reply:
Done https://github.com/pypa/setuptools_scm/pull/520/commits/c4d06b0c82b6f85233c715ff262d52bc3aa8bdf7 https://github.com/pypa/setuptools_scm/pull/520/commits/eae87b09a67699d10f63fceda3c47e8f371e8a1d